styles/components/profile.css

```diff
@@ -0,0 +1,86 @@
+.section-profile {
+  width: 100%;
+  text-align: center;
+}
+
+.img-profile {
+  display: block;
+  height: 180px;
```

Comment:
Dangerous - think of other font sizes or zooming.

Reply:
fixed to:
max-height: 180px;

just want to avoid, that the image becomes bigger. hope that works?